content/v2.0/reference/release-notes/flux.md

```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
 InfluxDB until the next InfluxDB v2.0 release._
 {{% /note %}}
 
+## v0.68.0 [2020-05-28]
+This version of Flux introduces an updated type inference system that improves
+performance, error messaging, and usability of the
+[Flux Language Server Protocol (LSP)](https://github.com/influxdata/flux-lsp).
+
+## Breaking Changes
+- Change signature of `group()` function.
+
+## Features
+- Add [`fieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys/) and
+  [`measurementFieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/measurementfieldkeys/)
+  to v1 package.
+- Add a context to `plantest.RuleTestCase`.
+- Add Snowflake support to SQL package.
+- Add `experimental.chain()` function.
```

Comment:
link to experimental.chain function?

Reply:
Docs for the function don't exist yet. Still waiting on feedback from the engineers.